core/src/main/scala/offheap/jni/JNILoader.java

```diff
@@ -0,0 +1,53 @@
+package offheap.jni;
```

Comment:
Our binding strategy is just an implementation detail and thus should rather be put under `offheap.internal` as we won't guarantee stability of this part of API.